src/services/review/ReviewRequestService.ts

```diff
@@ -125,6 +126,29 @@ export default class ReviewRequestService {
     return mappings
   }
 
+  computeCommentDataMappings = async (
```

Comment:
wdyt about renaming this to `computeCommentData` as it returns an array, not a map of x to y (eg: object/map)

Reply:
Sure, bd98768